core/src/main/resources/hudson/PluginManager/available.jelly

```diff
@@ -57,7 +57,7 @@ THE SOFTWARE.
           <thead>
             <tr>
               <l:isAdmin>
-                <th width="32">${%Install}</th>
+                <th data-sort-disable="true" />
```

Comment:
This used to tell users what the checkbox means. Now there's nothing left. Why?

Reply:
I think considering the average Jenkins user tends to be more of a power user the label isn't particularly beneficial.

My eventual plan is to move the select all button to its position, ala Gmail - 
<img width="66" alt="image" src="https://user-images.githubusercontent.com/43062514/166101484-c5adbb85-73b8-4119-9075-e19cad1c7c38.png">
And have the Select all/compatible/none be a dropdown from there.

Reply:
I expect this will potentially confuse new admins (IIRC there's not even tooltips). What's the advantage of having no label?

Reply:
I personally think the 'Install' column header is confusing, if it were to have a header ideally it would be 'Select' as that's what the column actually does, the install button is completely separate to the table. Plus I think it looks cleaner without it.

Anyhoo, I've restored it 👍